docs/sources/release-notes/v2-9.md

```diff
@@ -49,3 +49,11 @@ For a full list of all changes and fixes, look at the [CHANGELOG](https://github
 * Fix querying ingester for label values with a matcher (previously didn't respect the matcher).
 * Ensure all lifecycler cfgs ref a valid IPv6 addr and port combination
 
+### 2.9.4 (2024-01-24)

```

Comment:
nit: I know that we were already listing the different versions in increasing order (ex: first 2.9.1, then 2.9.2, etc), but I think the user experience is better if more recent releases are listed first. So 2.9.4, then 2.9.3, etc

Reply:
While I tend to agree, especially given the fact that in the 2.8.x branch the newest patch versions were first, see https://github.com/grafana/loki/blob/ned/release-v2.8.8/docs/sources/release-notes/v2-8.md :confused: 

I thought that someone had intentionally flipped the order for 2.9.x and so I didn't comment :sweat_smile:

Reply:
thanks for catching, i actually didn't realised. 
i changed the order to be reverse.